### ISSUE TITLE
Unable to use CreateUploadSettings

### ISSUE THREAD

**Issue by mabead** [SharpeRAD/Cake.AWS.S3#27]
I have a cake script that looks like this:

```
#addin nuget:?package=AWSSDK.Core&version=3.3.101.9
#addin nuget:?package=AWSSDK.S3&version=3.3.101.9
#addin "Cake.AWS.S3&version=0.6.8"

// ... and eventually
var uploadSettings = CreateUploadSettings();
```
As you can see, I am trying to call [CreateUploadSettings](https://www.cakebuild.net/api/Cake.AWS.S3/CakeContextExtensions/4A9A2F9C).

Unfortunately, it fails with this message:

```
error CS0103: The name 'CreateUploadSettings' does not exist in the current context
```

Do you know what I am doing wrong? 

My goal of using `CreateUploadSettings` is to use my aws credentials that are stored in my [aws credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html). Could you please also let me know if this will take these credentials?

I am trying to setup this so that I don't have to set the `AccessKey` and `SecretKey` in my `UploadSettings`. I would expect `Cake.AWS.S3` to pick these credentials automatically just like the aws cli does.

**Comment by eheywoodlonsdale** [comment]
@mabead 
Have you tried running Context.CreateUploadSettings() ? As the method is an extension method on Context
Ed

**Comment by mabead** [comment]
@eheywoodlonsdale The following does work!

```csharp
#addin nuget:?package=AWSSDK.Core&version=3.3.101.9
#addin nuget:?package=AWSSDK.S3&version=3.3.101.9
#addin "Cake.AWS.S3&version=0.6.8"

Task("foo")
    .Does(() => {
        var uploadSettings = Context.CreateUploadSettings();
    });
```
I just find it very unexpected that I have to use `Context.` in front of the function. Other extensions methods that have the exact same signature (ex: [DotNetCoreBuildServerShutdown(ICakeContext)](https://cakebuild.net/api/Cake.Common.Tools.DotNetCore/DotNetCoreAliases/AE02CC23)) can be called directly without having to use `Context.`. 

Do you understand why the `CreateUploadSettings` must be preceded by `Context.` and not other extension methods?

**Comment by SharpeRAD** [comment]
Fixed in v0.6.9 👍
